apps/dashboard/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@novu/dashboard",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
is there an option to skip updating the private packages?

Reply:
I will have a look.